digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/ProhibitionService.scala

```diff
@@ -68,16 +90,16 @@ class ProhibitionService(roadLinkServiceImpl: RoadLinkService, eventBusImpl: Dig
     val timing = System.currentTimeMillis
 
     val (assetsOnChangedLinks, assetsWithoutChangedLinks) = existingAssets.partition(a => LinearAssetUtils.newChangeInfoDetected(a, mappedChanges))
-
+    val myval = roadLinks.sortBy(_.linkId).headOption
```

Comment:
Accidently added to teach me one thing. Can be removed, since it's not even being used.

Reply:
would not be needed if we update the Scala version :)